src/com/goide/refactor/GoRenameImportSpecProcessor.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2013-2015 Sergey Ignatov, Alexander Zolotov, Florin Patan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.goide.refactor;
+
+import com.goide.psi.GoImportSpec;
+import com.goide.psi.impl.GoPsiImplUtil;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.psi.PsiElement;
+import com.intellij.refactoring.rename.RenamePsiElementProcessor;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class GoRenameImportSpecProcessor extends RenamePsiElementProcessor {
```

Comment:
To handle a dot import, couldn't find other way to prevent it from starting renaming and getting exception in MemberInplaceRenamer.
A possible solution is to resolve a dot symbol *itself* to the import path directory but I don't like the idea and seems like it will lead to another overriding named element logic for import specs.